public/app/plugins/datasource/elasticsearch/elastic_response.ts

```diff
@@ -310,11 +310,15 @@ export class ElasticResponse {
   }
 
   processHits(hits, seriesList) {
+    const hitsTotal = typeof hits.total === 'number' ? hits.total : hits.total.value; // <- Works with Elasticsearch 7.0+
+
     const series = {
       target: 'docs',
       type: 'docs',
       datapoints: [],
-      total: hits.total,
+      // XXX: `hits.total` is an object in ES 7, not a number
+      // ... leaving comment while trying to determine where this is called ...
+      total: hitsTotal,
```

Comment:
You can remove this comment and add a test instead. There's a test named `Raw documents query` in elastic_response.test.ts that you can copy and change to use es7 instead.

Reply:
Done, commit b3b337151b089544930c72942634f3a3c8f7123a